code/modules/trade/datums/trade_stations_presets/1-common/material_refine.dm

```diff
@@ -12,9 +12,9 @@
 	offer_limit = 30
 	base_income = 0		// Needs ore to refine
 	wealth = 0
-	hidden_inv_threshold = 2000
-	recommendation_threshold = 4000
-	stations_recommended = list("oddities")
+	hidden_inv_threshold = 3000
```

Comment:
Its kinda dumb that the Recoll has a hidden inventory threshold at all, honestly.

Reply:
Why the hell is it that I click "Add comment" instead of "Review" but it gets listed as a review anyways?

Reply:
> Why the hell is it that I click "Add comment" instead of "Review" but it gets listed as a review anyways?

Ignore it. It is a comment, we understand, GitHub just has funny language.